meteor-app/imports/ui/pages/for-employers/for-employers.jsx

```diff
@@ -13,12 +13,32 @@ const T = i18n.createComponent(t);
 
 const vizeBlue = "#2699FB";
 const vizePaleBlue = "#F1F9FF";
+const horizontalPaddingVal = "15px";
+
+const sizes = {
```

Comment:
@Mahfouzer Let's move this from the component to a config file. Also the colors above and the `horizontalPaddingVal`, all should be part of a config file not the component.

Reply:
Go ahead and merge from `master`. On master the colors are defined by a theme in `/meteor-app/imports/ui/theme.js` and responsive helpers have been defined in `/meteor-app/imports/ui/responsive.js`.